core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/cluster/ClusterV4Response.java

```diff
@@ -104,6 +104,12 @@
     @JsonDeserialize(using = Base64Deserializer.class)
     private String extendedBlueprintText;
 
+    @ApiModelProperty(StackModelDescription.CUSTOM_CONFIGURATIONS)
```

Comment:
should this and line #110 line have a different property description ?

Reply:
Acknowledged, will update model description to have name and crn of custom configs